cmd/exo/exo.rb

```diff
@@ -0,0 +1,10 @@
+cask 'exo' do
+  version '0.10.0a4'
```

Comment:
Where does the version come from? Shouldn't it be dynamic?

Reply:
It's was WIP (will be done with goreleaser)